strawberryfields/ops.py

```diff
@@ -463,15 +456,17 @@ def _apply(self, reg, backend, **kwargs):
             reg (Sequence[int]): subsystem indices the operation is
                 acting on (this is how the backend API wants them)
             backend (BaseBackend): backend to execute the operation
+
+        Returns:
+            array[Number] or None: Measurement results, if any. shape == (len(reg), shots).
```

Comment:
```suggestion
            array[Number] or None: Measurement results, if any; shape == (len(reg), shots).
```

Reply:
Fixed. BTW this isn't implemented yet, since the backends cannot handle multiple shots, yet.